go.mod

```diff
@@ -170,6 +170,7 @@ require (
 	github.com/knadh/koanf/v2 v2.0.1 // indirect
 	github.com/kr/pretty v0.3.1 // indirect
 	github.com/kr/text v0.2.0 // indirect
+	github.com/laher/mergefs v0.1.1
```

Comment:
I quickly checked the code, and this lib  does not recursively merge directories. We have a merge helper that does this, I guess it would be a better fit?
https://github.com/ory/x/blob/d50f3ce3b4dabde485cbe4bbf86ffb955493e7ce/fsx/merge.go

Reply:
Neither our own implementation (`fsx.Merge`) nor `mergefs.Merge` have perfect recursive merging. See this failing test https://github.com/ory/x/compare/master...merge-test
That's unfortunate but feels like "fixing" either would be a substantial amount of work for limited benefit.

We use laher/mergefs in Kratos already. It has a much smaller implementation than `fsx.Merge`. WDYT?

Reply:
fixed!